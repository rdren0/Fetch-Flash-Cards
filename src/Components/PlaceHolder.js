import React from 'react';
import Bulb from '../Images/bulb.svg'

function PlaceHolder(props) {
  return (
    <div className="card">
      <h2>Select a category to begin!</h2>
      <img className="bulb" src={Bulb} alt="Light Bulb palceholder"/>
    </div>
  );
}
  


export default PlaceHolder;
