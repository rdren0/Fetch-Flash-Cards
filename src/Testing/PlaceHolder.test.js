import PlaceHolder from '../Components/PlaceHolder';
import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });



describe('PlaceHolder', () => {
  let wrapper;
  
  beforeEach(() =>{
    wrapper = shallow(
      <PlaceHolder/>
    )
  })

  it('should match the snapshot with all data passed in correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});