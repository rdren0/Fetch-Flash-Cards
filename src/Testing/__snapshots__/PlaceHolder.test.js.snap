// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlaceHolder should match the snapshot with all data passed in correctly 1`] = `
<div
  className="card"
>
  <h2>
    Select a category to begin!
  </h2>
  <img
    alt="Light Bulb palceholder"
    className="bulb"
    src="bulb.svg"
  />
</div>
`;
